string vehicleName
geometry_msgs/PoseStamped reqVehPose

string[] blockedVehicles
---
bool removalStarted
